template/en/default/global/header.html.tmpl

```diff
@@ -380,12 +380,12 @@
 
 [%# in most cases the "header" variable provides redundant information, however
   # there are exceptions where not displaying this text is problematic. %]
-[% IF (user.settings.skin.value == 'Mozilla' || user.settings.skin.value == 'Mozilla-OpenSans')
+/*[% IF (user.settings.skin.value == 'Mozilla' || user.settings.skin.value == 'Mozilla-OpenSans')
```

Comment:
leave the IF, but remove the (user.settings.skin.value == 'Mozilla' || user.settings.skin.value == 'Mozilla-OpenSans') and the && below